app/src/main/java/com/blogspot/e_kanivets/moneytracker/activity/account/edit/fragment/AccountOperationsFragment.kt

```diff
@@ -73,6 +92,7 @@ class AccountOperationsFragment : BaseFragment() {
     companion object {
 
         private const val KEY_ACCOUNT = "key_account"
+        private const val EMPTY_DATE = "empty_date"
```

Comment:
I think, there is no need for this constant. You can just assign it in place or use `null`.